sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -278,6 +162,7 @@ module_platform_driver(skl_hda_audio)
 /* Module information */
 MODULE_DESCRIPTION("SKL/KBL/BXT/APL HDA Generic Machine driver");
 MODULE_AUTHOR("Rakesh Ughreja <rakesh.a.ughreja@intel.com>");
+MODULE_AUTHOR("Brent Lu <brent.lu@intel.com>");
```

Comment:
My take is that references to 'author' should be avoided, be it in a file header or module description. If I added this to all the files I touched I would be the author of pretty much everything in SOF. same for Ranjani, Peter, Kai, Bard. The commit tamper-proof Signed-off-by is enough to give you credit for your work.

Reply:
I just thought it is kinda mechanism that you could get notification when someone submit a patch to alsa-dev about it. I'll remove it from machine drivers if I got a chance.